deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+##############################################################################
+# deploy-kiali-to-kubernetes.sh
+#
+# This script deploys the Kiali components into kubernetes.
+#
+# To customize the behavior of this script, you can set one or more of the
+# environment variables used by this script.
+# See below for all the environment variables used.
+#
+# This script assumes all the kubernetes YAML files exist in the same
+# directory where this script is found. If an expected YAML file is missing,
+# an attempt will be made to download it.
+#
+# To use this script, the "oc" command must be in your PATH.
+# This script utilizes "envsubst" - make sure that command line tool
+# is installed and in your PATH.
+##############################################################################
+
+# The credentials can be specified either as already base64 encoded, or in plain text.
+# If the username or passphrase plain text variable is set but empty, the user will be asked for a value.
+if [ "${KIALI_USERNAME_BASE64}" == "" ]; then
+  KIALI_USERNAME="${KIALI_USERNAME=admin}" # note: the "=" inside ${} is on purpose
+  if [ "$KIALI_USERNAME" == "" ]; then
+    KIALI_USERNAME=$(read -p 'What do you want to use for the Kiali Username: ' val && echo -n $val)
+  fi
+  KIALI_USERNAME_BASE64="$(echo -n ${KIALI_USERNAME} | base64)"
+fi
+
+if [ "${KIALI_PASSPHRASE_BASE64}" == "" ]; then
+  KIALI_PASSPHRASE="${KIALI_PASSPHRASE=admin}" # note: the "=" inside ${} is on purpose
+  if [ "$KIALI_PASSPHRASE" == "" ]; then
+    KIALI_PASSPHRASE=$(read -sp 'What do you want to use for the Kiali Passphrase: ' val && echo -n $val)
+    echo
+  fi
+  KIALI_PASSPHRASE_BASE64="$(echo -n ${KIALI_PASSPHRASE} | base64)"
+fi
+
+export IMAGE_NAME="${IMAGE_NAME:-kiali/kiali}"
+export IMAGE_VERSION="${IMAGE_VERSION:-v0.10.0}"
```

Comment:
```suggestion
export IMAGE_VERSION="${IMAGE_VERSION:-v0.10}"
```
Default to "v0.10" so it picks up our latest patch version. speaking of which, I need this PR approved/merged so it makes the same change: https://github.com/kiali/kiali/pull/663